.github/workflows/integration-tests-dev.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       # This will use the cache for browsers. But it will reinstall system deps, which can't be cached
       # https://github.com/microsoft/playwright/issues/20603#issuecomment-1416243810
-      - run: pnpm dlx playwright install --with-deps
+      - run: pnpm dlx playwright@1.34.0 install --with-deps
```

Comment:
this fixes the playwright dependency mismatch

Reply:
I propose we use the less-pretty fix of forcing which binary to use in https://github.com/vercel/storage/pull/183/files, so versioning only happens at the package.json level

Reply:
I'll rebase it once merged